documentation/docs/development/workflow.md

```
@@ -0,0 +1,358 @@
+# Copy the HTTPS URL (e.g., https://abc123.ngrok-free.app)
+```
+
+**Terminal 2: Start WebSocket Server**
+```bash
+cd websocket-server
+# Update .env with your ngrok URL
+echo "PUBLIC_URL=https://abc123.ngrok-free.app" >> .env
+npm run dev
+```
+
+**Terminal 3: Start Webapp**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)